classes/requests/helpers/class-nets-easy-checkout-helper.php

```diff
@@ -182,4 +185,75 @@ public static function get_phone_number( $order ) {
 		}
 		return $phone_number;
 	}
+
+
+	/**
+	 * Gets customer address.
+	 *
+	 * @return array
+	 */
+	protected static function get_em_customer_data() {
+		$consumer          = array();
+		$email             = ! empty( WC()->checkout()->get_value( 'shipping_email' ) ) ? WC()->checkout()->get_value( 'shipping_email' ) : WC()->checkout()->get_value( 'billing_email' );
+		$consumer['email'] = $email;
+
+		$consumer['shippingAddress']['addressLine1'] = WC()->checkout()->get_value( 'shipping_address_1' );
```

Comment:
I think we should be consistent in if _billing_ of _shipping_ address should be used as the prioritized data. In some cases we default to shipping and in other to billing address. Maybe the function `get_consumer_address` 
 can be improved in the future but the logic there is:

```
if ( $order->get_billing_city() ) {
	$consumer['shippingAddress']['city'] = $order->get_billing_city();
}
```